docs/release-notes/v2.0.0.md

```diff
@@ -0,0 +1,24 @@
+## API Changes and Discord Integration
+
+**Breaking change**: the previous input parameter `SLACK_TOKEN` and `CHANNEL`
+have been deprecated and cannot be used anymore. Use `SLACK_WEBHOOK_URL` and
```

Comment:
If they can't be used any more, they're not "deprecated", they're just gone.

Reply:
ok, done.